deploy/clowdapp.yaml

```diff
@@ -572,3 +580,6 @@ parameters:
   - description: Notification service enabled
     name: APP_NOTIFICATIONS_ENABLED
     value: "true"
+  - description: Kafka TLS connection
+    name: KAFKA_TLS_ENABLED
+    value: "false"
```

Comment:
Review carefully... :-)

Reply:
I tend to write app options to be developer friendly. Many of other flags are sort of "run the app in standalone mode".

Reply:
Also note this is ephemeral friendly, for prod/stage I filed an MR to enable these explicitly.

Reply:
oki, it's just bit more prone for mistakes in prod, but it's fine :)
Worst case scenario, we would not be allowed to connect to kafka without TLS :shrug: :+1: